src/mon/FSCommands.cc

```diff
@@ -1525,6 +1513,32 @@ int FileSystemCommandHandler::_check_pool(
   }
 
   const string& pool_name = osd_map.get_pool_name(pool_id);
+  auto app_map = pool->application_metadata;
```

Comment:
minor nit for this code block: introduce a space after `if`.

Reply:
Done